payroll-processor-functions/src/payroll-processor-functions/Startup.cs

```diff
@@ -0,0 +1,17 @@
+using Microsoft.Azure.Functions.Extensions.DependencyInjection;
+using Newtonsoft.Json;
+using PayrollProcessor.Functions;
+using PayrollProcessor.Functions.Infrastructure;
+
+[assembly: FunctionsStartup(typeof(Startup))]
+
+namespace PayrollProcessor.Functions
+{
+    public class Startup : FunctionsStartup
```

Comment:
We don't currently have much need for function DI, but we can use this class to set that up when we do.